examples/plot_interpolate_bad_channels.py

```diff
@@ -51,7 +49,7 @@
 evoked_before.plot(exclude=[])
 
 # compute interpolation (also works with Raw and Epochs objects)
-evoked_after = evoked_before.copy().interpolate_bads_eeg()
+evoked_after = evoked_before.interpolate_bads()
 
-# plot interpolated (prevsious bads)
+# plot interpolated (previous bads)
 evoked_after.plot(exclude=[])
```

Comment:
>
> yes, the interpolated channels are still marked as bad
>
>
we should have a param to define this behavior. Basically otherwise the new
channel will never be used anywhere. What we may want to avoid is using
interpolated channels for inverse modeling but besides this use them (eg in
stats)